tests/NLog.UnitTests/Common/InternalLoggerTests.cs

```diff
@@ -38,6 +38,7 @@
 using Xunit;
 using NLog.Common;
 using System.Text;
+using Xunit.Extensions;
```

Comment:
CreateDirectoriesIfNeededTests is already inside an `#if !SILVERLIGHT` at [line 174](https://github.com/eduardorascon/NLog/blob/fix-create-directories/tests/NLog.UnitTests/Common/InternalLoggerTests.cs#L174).